modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/snapshot/SnapshotService.java

```diff
@@ -51,11 +48,12 @@ public void addSnapshotable(Snapshotable snapshotable) {
         } finally {
             executionPlanContext.getSharedLock().unlock();
         }
-        if (log.isDebugEnabled()) {
-            log.debug("Taking snapshot finished.");
-        }
+        log.debug("Taking snapshot finished.");
```

Comment:
Thanks @nirmal070125, change is done in pull request - https://github.com/wso2/siddhi/pull/73